node/_tools/suites/parallel/test-readline.js

```diff
@@ -0,0 +1,158 @@
+// deno-fmt-ignore-file
+// deno-lint-ignore-file
+
+// Copyright Joyent and Node contributors. All rights reserved. MIT license.
+// Taken from Node 16.13.0
+// This file is automatically generated by "node/_tools/setup.ts". Do not modify this file manually
+
+'use strict';
+const common = require('../common');
+const { PassThrough } = require('stream');
+const readline = require('readline');
+const assert = require('assert');
+
+common.skipIfDumbTerminal();
+
+{
+  const input = new PassThrough();
+  const rl = readline.createInterface({
+    terminal: true,
+    input: input
+  });
+  
+  rl.on('line', common.mustCall((data) => {
+    assert.strictEqual(data, 'abc');
+  }));
+  
+  input.end('abc');
+}
```

Comment:
These tests are not reliable right, eg. if I change `assert.strictEqual(data, 'abc');` to `assert.strictEqual(data, 'abcdef');` no error is thrown and tests still pass. Most likely related to #1482

Reply:
Or maybe it doesn't fail because I disabled catching of error in `events.ts`

Reply:
Yeah, definitely related to commenting out `events.ts`